src/index.css

```diff
@@ -2,3 +2,371 @@
   min-height: 100vh;
   height: 100vh;
 }
+
+
+head,
+body {
+  background-color: #ecf0f5;
+}
+.content-wrapper {
+  min-height: initial !important;
+}
+.bg-auth {
```

Comment:
Great thanks. I won't be able to work on this at all before Tuesday (again) but thanks for the comments.